### PR TITLE
Update release notes from Docs for 7.3 GA

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

This PR updates release notes from the Docs repo for 7.3 GA and edits to the other versions that have been made since the last version of Get-WhatsNew.

### CODE REVIEW

File: Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-70.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 description: New features and changes released in PowerShell 7.0
-ms.date: 10/22/2021
-title: What's New in PowerShell 7.0
+ms.date: 11/16/2022
```
Comment (adityapatwardhan) [inline Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-70.md:3]: Should this be today 11/29?

File: Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-72.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: What's New in PowerShell 7.2
 description: New features and changes released in PowerShell 7.2
-ms.date: 10/27/2021
+ms.date: 11/16/2022
```
Comment (adityapatwardhan) [inline Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-72.md:4]: Same as above

**Review by adityapatwardhan — APPROVED** [review]

File: Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-70.md (hunk 1 above)
Comment (sdwheeler) [reply Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-70.md:3]: No, that is when the file was last updated.

File: Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-72.md (hunk 2 above)
Comment (sdwheeler) [reply Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-72.md:4]: No, that is when the file was last updated.

**Pull request merged**